internal/app/handlers/shorturl.go

```diff
@@ -0,0 +1,144 @@
+package handlers
+
+import (
+	"encoding/base64"
+	"io"
+	"log"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+)
+
+const (
+	prefixURL string = "http://127.0.0.1:8080/"
+	sufixURL  string = ""
+)
+
+var listURL map[string]string
+
+func init() {
+	listURL = make(map[string]string)
+}
+
+func shortURL(url string) string {
+
+	minlong := 4
+
+	strval := []byte(url)
+	newURL := base64.StdEncoding.EncodeToString([]byte(strval))
+	if len(newURL) == minlong {
```

Comment:
```suggestion
	if len(newURL) <= minlong {
```
иначе при длине < 7 символов на else вылетит паника